### PR TITLE
chore: update docusaurus folder

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR copies over the fresh `misc/docusaurus` folder from the monorepo.

### CODE REVIEW

File: docusaurus/.gitignore [hunk 1]
```diff
@@ -18,4 +18,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-.yarn
\ No newline at end of file
+.yarn
+
+../.idea
```
Comment (albttx) [inline docusaurus/.gitignore:23]: useless

File: docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -61,12 +63,27 @@ const config = {
...
           },
+          {
+            position: "left",
+            label: "Playground",
+            to: "https://play.gno.land",
+          },
+          {
+            position: "left",
+            label: "Blog",
+            to: "https://test3.gno.land/r/gnoland/blog",
```
Comment (albttx) [inline docusaurus/docusaurus.config.js:85]: Don't we use portal loop ? (could be a future pr to update this)

**Review by albttx — COMMENTED** [review]
LGTM

File: docusaurus/docusaurus.config.js (hunk 2 above)
Comment (leohhhn) [reply docusaurus/docusaurus.config.js]: This was outdated. Fixed here and will fix it on the monorepo as well to match the change: [62e1ea3](https://github.com/gnolang/docs.gno.land/pull/4/commits/62e1ea3bd455f09083013e2d5eefb91fe12775bf)

File: docusaurus/.gitignore (hunk 1 above)
Comment (leohhhn) [reply docusaurus/.gitignore]: Fixed: [e98c996](https://github.com/gnolang/docs.gno.land/pull/4/commits/e98c99686eca9360129e80b4b8b9ba40bdc2cafc)

**Pull request merged**